enterprise/api/v1/views.py

```diff
@@ -999,6 +1000,37 @@ class EnterpriseCustomerBrandingConfigurationViewSet(EnterpriseReadOnlyModelView
     ordering_fields = FIELDS
     lookup_field = 'enterprise_customer__slug'
 
+    @action(methods=['patch'], detail=True, permission_classes=[permissions.IsAuthenticated])
+    @permission_required('enterprise.can_access_admin_dashboard', fn=lambda request, enterprise_uuid: enterprise_uuid)
+    def update_branding(self, request, enterprise_uuid):
+        """
+        POST /enterprise/api/v1/enterprise-customer-branding/update_branding
```

Comment:
this docstring says POST when its really a PATCH, and its missing the expected enterprise uuid path param.